{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"retryButton"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"UIButton","preciseIdentifier":"c:objc(cs)UIButton"},{"kind":"text","text":"! { "},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/talksdk\/callerrorview\/retrybutton"],"traits":[{"interfaceLanguage":"swift"}]},{"paths":["\/documentation\/talksdk\/callerrorview\/retrybutton"],"traits":[{"interfaceLanguage":"occ"}]}],"identifier":{"url":"doc:\/\/com.zendesk.TalkSDK\/documentation\/TalkSDK\/CallErrorView\/retryButton","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Retry button. When user will tap it call will try to reconnect."}],"kind":"symbol","metadata":{"navigatorTitle":[{"kind":"identifier","text":"retryButton"}],"role":"symbol","title":"retryButton","roleHeading":"Instance Property","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"retryButton"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"UIButton","preciseIdentifier":"c:objc(cs)UIButton"},{"kind":"text","text":"!"}],"symbolKind":"property","externalID":"c:@M@TalkSDK@objc(pl)CallErrorView(py)retryButton","required":true,"modules":[{"name":"TalkSDK"}]},"hierarchy":{"paths":[["doc:\/\/com.zendesk.TalkSDK\/documentation\/TalkSDK","doc:\/\/com.zendesk.TalkSDK\/documentation\/TalkSDK\/CallErrorView"]]},"references":{"doc://com.zendesk.TalkSDK/documentation/TalkSDK/CallErrorView":{"role":"symbol","title":"CallErrorView","fragments":[{"kind":"keyword","text":"protocol"},{"kind":"text","text":" "},{"kind":"identifier","text":"CallErrorView"}],"abstract":[{"type":"text","text":"Represent the error view with "},{"type":"codeVoice","code":"retry"},{"type":"text","text":" and "},{"type":"codeVoice","code":"cancel"},{"type":"text","text":" buttons."}],"identifier":"doc:\/\/com.zendesk.TalkSDK\/documentation\/TalkSDK\/CallErrorView","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"CallErrorView"}],"url":"\/documentation\/talksdk\/callerrorview"},"doc://com.zendesk.TalkSDK/documentation/TalkSDK/CallErrorView/retryButton":{"navigatorTitle":[{"kind":"identifier","text":"retryButton"}],"role":"symbol","title":"retryButton","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"retryButton"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"UIButton","preciseIdentifier":"c:objc(cs)UIButton"},{"kind":"text","text":"!"}],"abstract":[{"type":"text","text":"Retry button. When user will tap it call will try to reconnect."}],"identifier":"doc:\/\/com.zendesk.TalkSDK\/documentation\/TalkSDK\/CallErrorView\/retryButton","kind":"symbol","required":true,"type":"topic","url":"\/documentation\/talksdk\/callerrorview\/retrybutton"},"doc://com.zendesk.TalkSDK/documentation/TalkSDK":{"role":"collection","title":"TalkSDK","abstract":[],"identifier":"doc:\/\/com.zendesk.TalkSDK\/documentation\/TalkSDK","kind":"symbol","type":"topic","url":"\/documentation\/talksdk"}},"variantOverrides":[{"patch":[{"op":"replace","path":"\/identifier\/interfaceLanguage","value":"occ"},{"op":"replace","path":"\/metadata\/roleHeading","value":"Instance Property"},{"op":"replace","path":"\/metadata\/title","value":"retryButton"},{"op":"replace","path":"\/metadata\/symbolKind","value":"property"},{"op":"replace","path":"\/metadata\/fragments","value":[{"kind":"identifier","text":"retryButton"}]},{"op":"add","path":"\/topicSections","value":null},{"op":"add","path":"\/relationshipsSections","value":null},{"op":"add","path":"\/seeAlsoSections","value":null},{"op":"replace","path":"\/primaryContentSections\/0","value":{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"@property"},{"kind":"text","text":" ("},{"kind":"keyword","text":"nonatomic"},{"kind":"text","text":", "},{"kind":"keyword","text":"strong"},{"kind":"text","text":", "},{"kind":"keyword","text":"readonly"},{"kind":"text","text":") "},{"kind":"typeIdentifier","text":"UIButton","preciseIdentifier":"c:objc(cs)UIButton"},{"kind":"text","text":" * "},{"kind":"identifier","text":"retryButton"}],"languages":["occ"],"platforms":["iOS"]}]}},{"op":"replace","path":"\/references\/doc:~1~1com.zendesk.TalkSDK~1documentation~1TalkSDK~1CallErrorView\/title","value":"CallErrorView"},{"op":"replace","path":"\/references\/doc:~1~1com.zendesk.TalkSDK~1documentation~1TalkSDK~1CallErrorView\/fragments","value":[{"kind":"identifier","text":"CallErrorView"}]},{"op":"replace","path":"\/references\/doc:~1~1com.zendesk.TalkSDK~1documentation~1TalkSDK~1CallErrorView~1retryButton\/title","value":"retryButton"},{"op":"replace","path":"\/references\/doc:~1~1com.zendesk.TalkSDK~1documentation~1TalkSDK~1CallErrorView~1retryButton\/fragments","value":[{"kind":"identifier","text":"retryButton"}]}],"traits":[{"interfaceLanguage":"occ"}]}]}